demos/common/cpp/Dockerfile

```diff
@@ -101,7 +101,7 @@ RUN bazel build \
     @com_github_grpc_grpc//:grpc++ \
     @com_google_protobuf//:protobuf_lite \
     @org_tensorflow//tensorflow/core:lib \
-    @opencv//:opencv
+    @linux_opencv//:opencv
```

Comment:
client demos has a separate build configuration from the server so I guess we don't need to change it